site/includes/brand.hbs

```diff
@@ -1,6 +1,11 @@
 {{#unless provisional}}
 	<div class="brand col-xs-9 col-sm-5 col-md-4" property="publisher" resource="#wb-publisher" typeof="GovernmentOrganization">
+		{{#contains page.pageclass "home"}}
+		<img src="{{assets}}/../{{site.theme}}/assets/sig-blk-{{language}}.svg" alt="{{{i18n "tmpl-gc-sig"}}}" property="logo" /><span class="wb-inv"> / <span lang="{{#is page.language "en"}}fr">{{{i18n "tmpl-gc-sig" language="fr"}}}{{else}}en">{{{i18n "tmpl-gc-sig" language="en"}}}{{/is}}</span></span>
```

Comment:
FWIW this could be simplified and made less redundant by always using an ``<a>`` element and only using handlebars conditions to omit ``href="https://www.canada.ca/{{language}}.html" property="url"`` from the home page.

An ``<a>`` element with no ``href`` attribute represents a placeholder link in HTML (perfect fit for this kind of scenario - it's a placeholder for where a real link would've been if the user wasn't already on the page the link points to).

Reply:
Yeah, but in this scenario we are also adding a custom link to keep the URL for the publisher organisation

Reply:
We would prefer a pattern that would produce less ambiguity among accessibility expert, even if managing it would require us more effort because of complexing that handlebar includes.